.travis.yml

```diff
@@ -3,6 +3,14 @@ language: go
 go:
   - 1.x
   - 1.6
+  - master
+matrix:
+  allow_failures:
+    - go: master
+  fast_finish: true
+install:
+  - # Do nothing. This is needed to prevent default install action "go get -t -v ./..." from happening here (it is intended for this package to have no dependencies other than the standard library).
 script:
-  - diff -u <(echo -n) <(gofmt -d .)
-  - go test -v ./...
+  - diff -u <(echo -n) <(gofmt -d -s .)
```

Comment:
I suspect that Travis will complain shortly because of the '-s' flag :)

Feel free to commit the simplified Go files in the same PR.